docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+services:
+  db:
+    image: postgres
+    volumes:
+      - ./tmp/db:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
+    ports:
+      - 5432:5432
+  web:
+    image: crimethinc:latest
+    ports:
+      - "3000:3000"
```

Comment:
Here, if you don't setup another docker-compose `service` to front the Crimethinc. Web app like `nginx`, `haproxy` or anything else that can be used as a reverse proxy, you'll need to expose the port `3000` to the port `443` of the host machine. But if you do so, it'll also mean you need to handle SSL certificate.